contributor/video-proposal.md

```diff
@@ -18,47 +18,50 @@ This file represents the text intended for the video proposal (as-described in [
 ## Description & Learning Promise
 
 ### What is this video about, and why is it important?
-_Write a paragraph that answers this question._
+The Contributor role is one the most central roles in InnerSource - without Contributors, there would not be anything to receive or discuss about. 
+There's far more to it than just dumping some stuff in a branch or issue.
```

Comment:
```suggestion
There's far more to it than just dumping some code in a branch.
```

Reply:
In the outline we do highlight that no contribution is too small, especially contributions to issues etc. Thus a contribution does not always have to be code, although often it will likely be, which is why I added it to that sentence - I'm very open for something else than "stuff".

Reply:
> very open for something else than "stuff"

content?